src/BinDeps.jl

```diff
@@ -48,9 +48,9 @@ module BinDeps
         if downloadcmd === nothing
             for download_engine in @windows? (:powershell, :curl, :wget, :fetch) : (:curl, :wget, :fetch)
                 if download_engine == :powershell
-                    checkcmd = `$download_engine -help`
-                else
-                    checkcmd = `$download_engine --help`
+                    checkcmd = `cmd $download_engine new-variable foo`
+                 else
+                    checkcmd = `cmd $download_engine --help`
```

Comment:
this is wrong on unix. also the change in indent on the preceding line is superfluous